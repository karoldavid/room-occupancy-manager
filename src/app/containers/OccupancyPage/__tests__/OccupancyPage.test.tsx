import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { OccupancyPage } from '../index';

const renderer = createRenderer();

describe('<OccupancyPage />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<OccupancyPage />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
