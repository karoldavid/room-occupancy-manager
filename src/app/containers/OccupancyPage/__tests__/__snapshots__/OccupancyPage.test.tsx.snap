// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OccupancyPage /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Occupancy Page
    </title>
    <meta
      content="Room Occupancy Manager"
      name="description"
    />
  </Helmet>
  <Layout>
    <PageHeader
      className="site-page-header"
      subTitle="Calculate the number of guests and revenue"
      title="Room Occupation Manager"
    />
    <Divider
      orientation="left"
      style={
        Object {
          "color": "#333",
          "fontWeight": "normal",
        }
      }
    />
    <Layout>
      <Content
        style={
          Object {
            "padding": "0 50px",
          }
        }
      >
        <Row
          gutter={16}
        >
          <Col
            className="gutter-row"
            span={8}
          >
            <AvailableRoomsForm
              onFinish={[Function]}
              title="Available Rooms"
            />
          </Col>
        </Row>
        <Divider
          orientation="left"
          style={
            Object {
              "color": "#333",
              "fontWeight": "normal",
            }
          }
        />
        <Row
          gutter={16}
        >
          <Col
            span={8}
          >
            <RevenueDetails
              data={
                Object {
                  "currency": "EURO",
                  "free": 0,
                  "total": 0,
                  "usage": 0,
                }
              }
              title="Premiun"
            />
          </Col>
          <Col
            span={8}
          >
            <RevenueDetails
              data={
                Object {
                  "currency": "EURO",
                  "free": 0,
                  "total": 0,
                  "usage": 0,
                }
              }
              title="Economy"
            />
          </Col>
        </Row>
      </Content>
    </Layout>
    <Footer>
      Credits
    </Footer>
  </Layout>
</React.Fragment>
`;
